armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py

```diff
@@ -425,11 +426,11 @@ def test_calcReactionRates(self):
         """
         Test that the reaction rate code executes and sets a param > 0.0.
 
+        TODO: This could validate the reaction rate calculation.
```

Comment:
Why change the warning to a TODO?

Reply:
Indeed.

It is a test, and this "message" says we need to improve the test.  That is not a "warning", but a call for more work.  So, I believe this is more accurately termed a "TODO".

Which matters, because I search for TODOs to complete them.  But this warning could be lost in the shuffle forever.

It is a bit off-topic for this PR, but when wouldn't it be off-topic?